.circleci/config.yml

```diff
@@ -47,20 +47,39 @@ var_6: &save_cache
 var_7: &yarn_install
   run: yarn install --frozen-lockfile --non-interactive
 
+# Copies the Bazel config which is specifically for CircleCI to a location where Bazel picks it
+# up and merges it with the project-wide bazel configuration (tools/bazel.rc)
+var_8: &copy_bazel_config
+  # Set up the CircleCI specific bazel configuration.
+  run: sudo cp ./.circleci/bazel.rc /etc/bazel.bazelrc
+
 # Sets up a different Docker image that includes a moe recent Firefox version which
 # is needed for headless testing.
-var_8: &docker-firefox-image
+var_9: &docker-firefox-image
   # TODO(devversion): Temporarily use a image that includes Firefox 62 because the
   # ngcontainer image does include an old Firefox version that does not support headless.
   # See the PR that fixes this: https://github.com/angular/angular/pull/26435
   - image: circleci/node:10.12-browsers
 
 # Attaches the release output which has been stored in the workspace to the current job.
 # https://circleci.com/docs/2.0/workflows/#using-workspaces-to-share-data-among-jobs
-var_9: &attach_release_output
+var_10: &attach_release_output
   attach_workspace:
     at: dist/releases
 
+# Branch filter that we can specify for jobs that should only run on publish branches. This filter
+# is used to ensure that not all upstream branches will be published as Github builds
+# (e.g. revert branches, feature branches)
+var_11: &publish_branches_filter
+  branches:
+    only:
+    - master
+    # 6.0.x, 7.1.x, etc.
+    - /\d+\.\d+\.x/
+    # 6.x, 7.x, 8.x etc
+    - /\d+\.x/
```

Comment:
Not needed as there are no publish branches here. Only `master` should be fine.

Reply:
We're going to add a 6.x branch in the future pending @ThomasBurleson